src/Handlers/DefaultHandler.php

```diff
@@ -15,6 +15,6 @@ class DefaultHandler extends BaseHandler
      */
     public function handle(array $params): ResponseInterface
     {
-        return new JsonApiResponse(IResponse::S400_BAD_REQUEST, ['status' => 'error', 'message' => 'Unknown api endpoint']);
+        return new JsonApiResponse(IResponse::S404_NotFound, ['status' => 'error', 'message' => 'Unknown api endpoint']);
```

Comment:
this constant is not available in older versions which are also supported by this extension

and unfortunatelly this is BC break - some apps can check status code 400 for unknown api endpoints (I was pretty sure I did this in version 2.0.0, but probably it get lost somewhere